examples/with-typescript/tslint.json

```diff
@@ -0,0 +1,10 @@
+{
+  "defaultSeverity": "error",
+  "extends": ["tslint:recommended"],
+  "jsRules": {},
+  "rules": {
+    "quotemark": [true, "single", "jsx-double"],
+    "semicolon": [true, "never"]
```

Comment:
I've changed default rules, to match next.js coding style (but I don't like it). Is it possible to stick to default rules (using double quote and always append semicolons).